src/utils/Configuration.js

```diff
@@ -0,0 +1,96 @@
+"use strict";
```

Comment:
This looks like a transpiled file. Why can't we use TS?

Reply:
Wrong file was originally committed. Fixed now.